### PR TITLE
adds ang search and romanized gurmukhi search

### PR DESCRIPTION
**Pull request by inderpreetsingh**
* Goes hand in hand with https://github.com/KhalisFoundation/sttm-core/pull/14
angsearch
romanizedgurmukhi

## Suggestions
* Phonetic search in romanized gurmukhi search.
* Separate select box for selecting source in search.

### CODE REVIEW

File: www/js/search.js [hunk 1]
```diff
@@ -38,21 +38,21 @@ module.exports = {
...
-        const words = searchQuery.split(' ');
-        dbQuery = `%${words.join(' %')}%`;
-        condition = `${searchCol} LIKE '${dbQuery}'`;
         break;
       }
       case 5: // Ang
         searchCol = 'PageNo';
         dbQuery = parseInt(searchQuery, 10);
-        condition = `${searchCol} = ${dbQuery} AND v.SourceID = '${global.platform.search.currentMeta.source}'`;
+        condition = `${searchCol} = ${dbQuery} AND v.SourceID = '${global.core.search.currentMeta.source || 'G'}'`;
```
Comment (inderpreetsingh) [inline www/js/search.js]: @navdeepsinghkhalsa I was not sure why `platform.search.currentMeta` was here. I could find `currentMeta` in core.search but it didn't have a source attribute. So I used 'G' as default source.

**Comment by navdeepsinghkhalsa** [comment]
@inderpreetsingh This is a good PR, but our technique for transliteration search is a bit different than just comparing to the transliteration column, because everybody has different spellings. We'll update this after merged codebase

**Comment by inderpreetsingh** [comment]
@navdeepsinghkhalsa Veerji, ha'nji was thinking the same. What do you think about using some phonetic library? I have had some success previously with phonetic search, when searching for romanized gurmukhi. 🙏 

Also, if you can point me to what technique we are using, I can try using that one in this PR.

**Comment by inderpreetsingh** [comment]
Closing this PR as ang search has been sent in another PR and romanized search is yet to be finalized.

**Pull request closed without merging**